src/js/va-letters/utils/helpers.js

```diff
@@ -1,4 +1,40 @@
+import merge from 'lodash/fp/merge';
+
+import environment from '../../common/helpers/environment';
+
 export const chapterNames = {
   confirmAddress: 'Confirm Address',
   downloadLetters: 'Download Letters'
 };
+
+function isJson(response) {
```

Comment:
Is this all copied from somewhere else? I noticed similar code in the rx app. If so, perhaps it's something we can extract out and share between apps?